### PR TITLE
Board flashing instructions updates for v89

### PR DESCRIPTION
**Pull request by ricardosalveti**
## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

Adds flashing support for NVIDIA Xavier and Orin developer kits, plus minor fixes found for other boards.

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [x] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 29ee50a4bc8acd73b21aa847836a065b11deee61 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1854/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 6f0452290ae91189bca7e9fe8526f81b05a26e21 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1855/docs/artifacts/html/index.html

File: source/reference-manual/boards/jetson-agx-orin-devkit.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+.. figure:: /_static/boards/jetson-agx-orin-devkit.png
+     :width: 600
+     :align: center
+
+     jetson-agx-orin-devkit
+
+1. Connect the USB cable in the base board Type-C connector (10) to the host machine.
+
+2. Press and hold down the Force Recovery button, then either press the Reset button (if already powered) or press the Power button, releasing both buttons in the end.
+
```
Comment (quaresmajose) [inline source/reference-manual/boards/jetson-agx-orin-devkit.rst:44]: To check if the board is in recovery mode `lsusb` can be used to check if the device `[vendor]:[product]` is available.
Where [vendor] is `0955` and the [product] is the device id, please note that this only appears on the `lsusb` output when in recovery mode.

**Comment by doanac** [comment]
Docs for 263e55d01a5134504bac165103a3d331feef1a95 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1858/docs/artifacts/html/index.html

File: source/reference-manual/boards/jetson-agx-orin-devkit.rst (hunk 1 above)
Comment (ricardosalveti) [reply source/reference-manual/boards/jetson-agx-orin-devkit.rst:44]: Added for both orin and xavier, thanks!

**Pull request merged**